packages/fx-core/src/common/tools.ts

```diff
@@ -90,18 +119,29 @@ export function objectToConfigMap(o?: Json): ConfigMap {
 }
 
 const SecretDataMatchers = [
+  "solution.localDebugTeamsAppId",
+  "solution.teamsAppTenantId",
   "fx-resource-aad-app-for-teams.clientSecret",
   "fx-resource-aad-app-for-teams.local_clientSecret",
+  "fx-resource-aad-app-for-teams.local_clientId",
```

Comment:
The naming seems not consistent, some parts of the name using “local_" and the other use "localXXXX"， is there a reason for that?

Reply:
For some reason some plugins did not follow "local_" type. We may fix this when moving to Arm.